vision/google/cloud/vision/_http.py

```diff
@@ -41,20 +41,20 @@ def annotate(self, image, features):
                          based on the number of Feature Types.
 
                          See: https://cloud.google.com/vision/docs/pricing
-        :rtype: dict
-        :returns: List of annotations.
+        :rtype: list or class:`~googe.cloud.vision.annotations.Annotations`
+        :returns: Instance of ``Annotations`` or List of ``Annotations``.
         """
         request = _make_request(image, features)
 
         data = {'requests': [request]}
         api_response = self._connection.api_request(
             method='POST', path='/images:annotate', data=data)
-        images = api_response.get('responses')
-        if len(images) == 1:
```

Comment:
Same comment as above. Return a list consistently or object consistently, but do not magic-flatten for a single result.